app/Dockerfile

```diff
@@ -19,36 +19,36 @@ FROM --platform=$TARGETPLATFORM python:3.9-slim-bullseye as builder
 
 RUN apt-get update && apt-get install -y binutils
 
-COPY ./src /src
+COPY ./app /app
```

Comment:
Fix the SDK app structure to match the template app/src/.. . Reason: 
- after importing the seat example, deploying the vehicle app to the k3d fails due to